src/Kernel/Object.class.st

```diff
@@ -894,9 +894,9 @@ Object >> explicitRequirement [
 	| originalMethod originalArguments errorBlock originalReceiver callingContext originalSelector |
 	errorBlock := [ ^ self error: 'Explicitly required method' ].
 	callingContext := thisContext sender.
-	originalMethod := callingContext compiledCode method.
+	originalMethod := callingContext homeMethod.
 	originalMethod isFromTrait
-		ifFalse: errorBlock.
+		ifFalse: [ errorBlock value].
```

Comment:
For curiosity, why this change?

Reply:
just to use the optized ifFalse... I guess not important here as speed does not matter